synadm/cli/__init__.py

```diff
@@ -62,6 +75,7 @@ class APIHelper:
     FORMATTERS = {
         "pprint": pprint.pformat,
         "json": json_pretty,
+        "minified": json_minify,
```

Comment:
Not a requested change, but you *could* use lambdas instead. Just an idea.

I've already done it on 9a41943.